client/blocks/comment-button/index.jsx

```diff
@@ -38,34 +44,43 @@ const CommentButton = React.createClass( {
 
 	render() {
 		let label;
-		const containerTag = this.props.tagName,
-			commentCount = this.props.commentCount;
+		const { count, showLabel, size, tagName: containerTag } = this.props;
 
-		if ( commentCount === 0 ) {
+		if ( count === 0 ) {
 			label = this.translate( 'Comment' );
 		} else {
 			label = this.translate(
 				'Comment',
 				'Comments', {
-					count: commentCount,
+					count: count,
 				}
 			);
 		}
 
 		const labelElement = ( <span className="comment-button__label">
-			{ commentCount > 0 ? <span className="comment-button__label-count">{ commentCount }</span> : null }
-			{ this.props.showLabel && <span className="comment-button__label-status">{ label }</span> }
+			{ count > 0 ? <span className="comment-button__label-count">{ count }</span> : null }
+			{ showLabel && <span className="comment-button__label-status">{ label }</span> }
 		</span> );
 
 		return React.createElement(
 			containerTag, {
-				className: 'comment-button',
+				className: classNames( {
+					'comment-button': true,
```

Comment:
If a property of the object passed to `classNames` is always true, you can alternative pass it as a separate string argument, i.e.:

```js
className: classNames( 'comment-button', {
	'is-empty': count === 0
} )
```